pkg/generic/generic.go

```diff
@@ -20,6 +20,8 @@ package generic
 import (
 	"fmt"
 
+	"github.com/cloudwego/kitex/pkg/remote/codec/protobuf"
```

Comment:
We write the import to have three blocks in this order: standard package, external package, and internal package (= github.com/cloudwego/kitex/xxx). So, please delete L24 and sort alphabetically (in this case "github.com/cloudwego/kitex/pkg/remote/codec/protobuf" should be inserted between "github.com/cloudwego/kitex/pkg/remote" and "github.com/cloudwego/kitex/pkg/remote/codec/thrift").

Reply:
Same for other files.